specs/core/1_custody-game.md

```diff
@@ -369,15 +361,18 @@ def process_custody_key_reveal(state: BeaconState,
 
     # Decrement max reveal lateness if response is timely
     if revealer.next_custody_reveal_period == get_validators_custody_reveal_period(state, reveal.revealer_index) - 2:
-            revealer.max_reveal_lateness -=  MAX_REVEAL_LATENESS_DECREMENT
-    revealer.max_reveal_lateness = max(revealed_validator.max_reveal_lateness, get_validators_custody_reveal_period(state, reveal.revealed_index) - revealer.next_custody_reveal_period)
+        revealer.max_reveal_lateness -= MAX_REVEAL_LATENESS_DECREMENT
+    revealer.max_reveal_lateness = max(
+        revealer.max_reveal_lateness,
+        get_validators_custody_reveal_period(state, reveal.revealed_index) - revealer.next_custody_reveal_period
+    )
 
     # Process reveal
     revealer.next_custody_reveal_period += 1
 
     # Reward Block Preposer
     proposer_index = get_beacon_proposer_index(state)
-    increase_balance(state, proposer_index, base_reward(state, index) // MINOR_REWARD_QUOTIENT)
+    increase_balance(state, proposer_index, get_base_reward(state, proposer_index) // MINOR_REWARD_QUOTIENT)
```

Comment:
This reawrd might better be a function of the `revealer_index` rather than `proposer_index`

Reply:
Agreed